api/v1/reviews/PKPReviewController.php

```diff
@@ -225,4 +257,402 @@ public function getHistory(Request $illuminateRequest): JsonResponse
 
         return response()->json($reviewRoundHistory, Response::HTTP_OK);
     }
+
+    /**
+     * Export a review as PDF
+     */
+    public function exportReviewPDF(Request $illuminateRequest): void
+    {
+        $request = $this->getRequest();
+        $submissionCommentDao = DAORegistry::getDAO('SubmissionCommentDAO'); /* @var $submissionCommentDao SubmissionCommentDAO */
+        $context = $request->getContext();
+        $submissionId = $request->getUserVar('submissionId');
+        $reviewId = $request->getUserVar('reviewAssignmentId');
+        $reviewAssignment = Repo::reviewAssignment()->get($reviewId);
+        $submissionComments = $submissionCommentDao->getReviewerCommentsByReviewerId($submissionId, $reviewAssignment->getReviewerId(), $reviewId, true);
+        $submissionCommentsPrivate = $submissionCommentDao->getReviewerCommentsByReviewerId($submissionId, $reviewAssignment->getReviewerId(), $reviewId, false);
+        $submission = $this->getAuthorizedContextObject(Application::ASSOC_TYPE_SUBMISSION);
+        $title = $submission->getCurrentPublication()->getLocalizedTitle(null, 'html');
+        $cleanTitle = str_replace("&nbsp;", " ", strip_tags($title));
+        $mpdf = new Mpdf([
```

Comment:
To aid in readability of this and the XML method below, could you break out the actual PDF/XML generation into separate functions so the API handling vs the PDF/XML generation are a bit easier to follow and separate? If these endpoints/controllers grow in the future, I can see wanting to extract the document generation out to separate classes as well.